.github/workflows/spec.yml

```diff
@@ -0,0 +1,55 @@
+name: Specification
+
+on:
+  push:
+    tags:
+      - '*'
```

Comment:
Should it be run on the publish of RC tag?

Reply:
The idea is to publish on every merge, since it's still quite work-in-progress.